depthai_sdk/src/depthai_sdk/components/stereo_component.py

```diff
@@ -288,7 +293,12 @@ def set_colormap(self, colormap: dai.Colormap):
             colormap_manip = self.node.getParentPipeline().create(dai.node.ImageManip)
             colormap_manip.initialConfig.setColormap(colormap, self.node.initialConfig.getMaxDisparity())
             colormap_manip.initialConfig.setFrameType(dai.ImgFrame.Type.NV12)
-            h, w = self.left.stream_size
+            if self._align_component:
+                h, w = self._align_component.node.getVideoSize() \
```

Comment:
I think it's actually ISP not video (haven't tested tho)
```suggestion
                h, w = self._align_component.node.getIspSize() \
```